pkg/metrics/metrics.go

```diff
@@ -827,6 +847,26 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 			collectors = append(collectors, PolicyImplementationDelay)
 			c.PolicyImplementationDelayEnabled = true
 
+		case Namespace + "_cidr_group_translation_time_stats_seconds":
+			CIDRGroupTranslationTimeStats = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+				Namespace: Namespace,
+				Name:      "cidr_group_translation_time_stats_seconds",
+				Help:      "CIDRGroup translation time stats",
+			}, []string{LabelPolicyNamespace, LabelPolicyName})
```

Comment:
Hmm, I don't think we should be using policy names in metrics since that's user input and the cardinality can reach very high.

Reply:
Fixed with a histogram without additional labels.